cli/bin/cli.js

```diff
@@ -4,18 +4,21 @@ const open = require('open');
 const { showHelp, showVersion, showErrorMsg } = require('../index');
 
 async function main() {
+  const validFlags = ['-v', '--version', '-h', ' --help'];
   const message = process.argv[2];
 
-  if (message === undefined || message === '-h' || message === '--help') {
+  if (
+    message === undefined ||
+    message.toLowerCase() === '-h' ||
+    message.toLowerCase() === '--help'
+  ) {
     showHelp();
-  } else if (message === '-v' || message === '--version') {
-    showVersion();
   } else if (
-    (message.charAt(0) === '-' && message !== '-h') ||
-    (message.charAt(0) === '-' && message !== '--help') ||
-    (message.charAt(0) === '-' && message !== '-v') ||
-    (message.charAt(0) === '-' && message !== '--version')
+    message.toLowerCase() === '-v' ||
+    message.toLowerCase() === '--version'
   ) {
+    showVersion();
+  } else if (message.charAt(0) === '-' && !validFlags.includes(message)) {
```

Comment:
what about early return?

Reply:
This line of code will create a problem **not allowing people** to start a twitter message with a character being the "-" (minus)